packages/mdc-typography/_variables.scss

```diff
@@ -22,7 +22,7 @@
 
 @import "./functions";
 
-$mdc-typography-font-family: Roboto, sans-serif !default;
```

Comment:
Does our override mixins break if we set without quotes?

Reply:
Which override mixins? The `mdc-typography` mixins ?

Reply:
Yes, mixins / functions that use this variable.

Reply:
This variable is used below in the font styles map, but I would assume that our screenshot tests would break if the mixins/functions would break with this change.

Reply:
`unquote("Roboto, sans-serif")` seems to be same as `Roboto, sans-serif`. I looked at our typography mixins / functions, we don't seem to validate this type anywhere.

```sass
$fonts: unquote("Arial, sans-serif");
@debug type-of($fonts); // Return string

$fonts: Arial, sans-serif;
@debug type-of($fonts); // Return list

font-family: $fonts; // font-family can accept both list & string type.
```

Having said that this change looks good to me. But sass automatically type casts the list to string when applied to CSS property as mentioned above.

WDYT?

Reply:
I think either way works for me, as long as we're consistent in both what we recommend users use (via docs) and what we use.